packages/apib-parser/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Fury API Blueprint Parser Changelog
+# API Elements: API Blueprint Parser Changelog
+
+## Master
```

Comment:
❓ I was planning to release this as 0.18.1 so that the "same" version of this adapter can be used with both package names. Not sure what to do in changelog though.

Reply:
Perhaps its clearer if I make this as a `0.19.0` release with the new name.

Reply:
@opichals ok, i've pushed 541dd8e4 which bumps the version of this adapter to 0.19.0.